### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by dhodovsk**
Based on feedback provided here: https://github.com/container-images/vsftpd/issues/1

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ ENV SUMMARY="Very Secure Ftp Daemon" \
     RELEASE=1 \
     ARCH=x86_64
 
+LABEL MAINTAINER "Dominika Hodovska" <dhodovsk@redhat.com>
```
Comment (TomasTomecek) [inline Dockerfile]: I don't want to nitpick badly, but can you use lower case characters in the label name? (meaning `maintainer`)

File: README.md [hunk 2]
```diff
@@ -8,18 +8,53 @@ docker build -t vsftpd .
...
-docker run -p 20:20 -p 21:21 -p 21100-21110:21100-21110 \
+$ docker run -p 20:20 -p 21:21 -p 21100-21110:21100-21110 \
 --rm -v /etc/vsftpd/:/etc/vsftpd/ \
 -v <shared files directory>:/var/ftp/pub \
 --name vsftpd vsftpd
 ```
 
+## Running in passive mode
+
+To configuration add following lines:
```
Comment (TomasTomecek) [inline README.md]: `Add following lines to the configuration file:`

**Review by TomasTomecek — CHANGES_REQUESTED** [review]
Sweet! Just one nit and one grammar improvement.

**Comment by dhodovsk** [comment]
Changes should be there. Thank you for review! :)

**Review by TomasTomecek — APPROVED** [review]
Nicely done!

**Pull request merged**